roxie/ccd/ccdserver.cpp

```diff
@@ -15777,6 +15780,8 @@ class CRoxieServerParallelGraphLoopActivity : public CRoxieServerGraphLoopActivi
         resultStream = NULL;
         resultJunction.clear();
         outputs.kill();
+        ForEachItemIn(i, iterationGraphs)
```

Comment:
Bit nervous that this is not too late, given the comment on the line below?

Reply:
I agree.  I'll move it earlier.